### ISSUE TITLE
bug: Provider 1.6.1 throws error context deadline exceeded (Client.Timeout exceeded while awaiting headers)

### ISSUE THREAD

**Issue by bebold-jhr** [lacework/terraform-provider-lacework#479]
**Describe the bug**
We've got an internal PR updating the provider **from 1.5.1 to 1.6.1**. Our plan steps fail, because the provider throws the following error:

> Error: Post "https://####.lacework.net/api/v2/access/tokens": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

**To Reproduce**
1. Deploy with 1.5.1
2. Update provider to 1.6.1
3. terraform plan

**Expected behavior**
Runs plan successfully.

**Screenshots**

**Please complete the following information:**
 - **OS:** debian:bookworm-20230227-slim 
 - **Provider Version:** 1.6.1

**Additional context**
Happens in multiple different terraform modules. Error message is always the same.
Plan still works with 1.5.1 in the same modules.

**Comment by dmurray-lacework** [comment]
Hi @bebold-jhr I have not been able to reproduce this. Can you give anymore details to your test scenario. Here are the tests I've ran:

Provider
```
Set version to 1.5.1 for examples/resource_lacework_alert_channel_email/main.tf
Terraform init
Terraform apply
Update version to 1.6.1 & run Terraform init -upgrade 
Terraform plan
Terraform apply
```

Module
```
Checkout terraform-aws-config
Set version of provider to 1.5.1 for example/default_config/main.tf
Terraform init
Terraform apply
Update version to 1.6.1 & run Terraform init -upgrade 
Terraform plan
Terraform apply
```

**Comment by bebold-jhr** [comment]
Hi @dmurray-lacework,
thank you for your fast reply. I'll try to create a minimal example apart from the actual project code.

**Comment by dmurray-lacework** [comment]
@bebold-jhr I believe so yes, I've merged a change that should resolve your issue. The fix will be in the provider version `v1.6.2`

**Comment by bebold-jhr** [comment]
@dmurray-lacework looks good to me. The update PRs for 1.6.2 were able to run the plan successfully. Thank you for the quick fix.
